src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```diff
@@ -4,6 +4,7 @@
 parameters:
     env(SYLIUS_API_AUTHORIZATION_HEADER): Authorization
     sylius.api.authorization_header: "%env(resolve:SYLIUS_API_AUTHORIZATION_HEADER)%"
+    sylius.api.paths_to_hide: [ "/api/v2/admin/catalog-promotion-rules/{id}" ]
```

Comment:
I believe there is a rebase needed, as this parameters [has already been added](https://github.com/Sylius/Sylius/pull/13105/files#diff-af06379517d468a6c5db1fe5e1a16f0ff1191c95eb300ad02f96f1a418f1a9dbR7) 🚀